packages/components/notifications/src/content-notification/README.md

```diff
@@ -16,10 +16,11 @@ import { ContentNotification } from '@commercetools-uikit/notifications';
 
 ## Properties
 
-| Props         | Type           | Required | Values                                | Default | Description                                                          |
-| ------------- | -------------- | :------: | ------------------------------------- | ------- | -------------------------------------------------------------------- |
-| `type`        | `string`       |    ✅    | `success`, `info`, `warning`, `error` | -       | Type of notification, determining its appearance.                    |
-| `children`    | `node`         | ✅ (\*)  | -                                     | -       | Message to be shown inside a notification panel.                     |
-| `intlMessage` | `intl message` | ✅ (\*)  | -                                     | -       | An intl message object that will be rendered with `FormattedMessage` |
+| Props           | Type           | Required | Values                                | Default | Description                                                          |
+| --------------- | -------------- | :------: | ------------------------------------- | ------- | -------------------------------------------------------------------- |
+| `type`          | `string`       |    ✅    | `success`, `info`, `warning`, `error` | -       | Type of notification, determining its appearance.                    |
+| `containerType` | `string`       |          | `flex`, `block`                       | `flex`  | Type of container for children, determining its layout.              |
```

Comment:
MDN calls this a "display type". The CSS spec seems to differentiate between `display-inside` and `display-outside` values. `block` being a `display-outside` value and `flex` a `display-inside` value.

That brings me to two ideas:

1. `displayType` might be a better name
2. Are we just wanting to control row or column layout in the end?

I wonder if anybody ever assumed that you can layout two children inside the notification. So changing anything here isn't much breaking more of a bug fix and using `flex` and direction of sorts would be fine? Sorry for discussing it here again.

Reply:
from my side, `displayType` sounds more appropriate and we are trying to control only the row layout.